core/modules/system/system.updater.inc

```diff
@@ -3,7 +3,7 @@
 /**
  * @file
  * Subclasses of the Updater class to update Backdrop core knows how to update.
- * At this time, only modules and themes are supported.
+ * At this time, only modules, themes and layouts are supported.
```

Comment:
Could we add in an Oxford comma here and everywhere else? `modules, themes, and layouts`